pages/01.HDM Documentation/06.vcd/10.Network Planning/05.Cloud Network Config/default.md

```diff
@@ -12,29 +12,34 @@ This section, describes how to configure the VDC required for HDM deployment.
 
 This network will be connected to the edge gateway. Cross-WAN communications between on-premises and cloud will be routed through this network for the HDM Appliance. IPsec tunnels [step 3](#configure-vpn-tunnel-edge) will be configured to this network to complete the network configuration.
 
+1. Select the _Virtual Data Center_ that will be the target of migration.
+![Select ORG VDC](images/select-vdc.png?classes=content-img "Select Org VDC")
+
+1. Select _Network_ -> _NEW_ to create a new network.
+![Create new network](images/create-new-network.png?classes=content-img "Create new network")
 1. Create a new network and select Routed Network Type
 
-![alt_text](images/image4.png "image_tooltip")
+![alt_text](images/image4.png?classes=content-img "image_tooltip")
 
-2. Provide the name “UPLINK_WAN_NETWORK” and Gateway CIDR for the network. Note the CIDR; this will be required when creating the VPN connection between on-premises and the edge gateway in the cloud.
+1. Provide the name “UPLINK_WAN_NETWORK” and Gateway CIDR for the network. Note the CIDR; this will be required when creating the VPN connection between on-premises and the edge gateway in the cloud.
 
-![alt_text](images/image3.png "image_tooltip")
+![alt_text](images/image3.png?classes=content-img "image_tooltip")
 
-3. Select the edge gateway 
+1. Select the edge gateway 
```

Comment:
Counts are not starting with 1. Is this intentional?

Reply:
In mark down numbered lists are written as

```
1. item one
1. item two
```

This gets rendered at 

1. item one
1. item two

The only thing we need to take care of is the paras.